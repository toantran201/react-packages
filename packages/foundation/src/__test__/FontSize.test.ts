import FontSize from "../FontSize";

test('Snapshot of fontsize', () => {
	expect(FontSize).toMatchSnapshot()
})
