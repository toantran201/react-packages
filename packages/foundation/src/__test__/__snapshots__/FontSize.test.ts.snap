// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of fontsize 1`] = `
Object {
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
}
`;
